cmd/skyeye/main.go

```diff
@@ -81,6 +82,7 @@ func init() {
 	_ = skyeye.MarkFlagRequired("whisper-model")
 	voiceFlag := NewEnum(&voiceName, "Voice", "", "feminine", "masculine")
 	skyeye.Flags().Var(voiceFlag, "voice", "Voice to use for SRS transmissions (feminine, masculine)")
+	skyeye.Flags().Float32Var(&playbackSpeed, "voice-playback-speed", 1.0, "Voice playback speed of GCI")
```

Comment:
May want to hint user how to customize:

"Speed at which SRS transmissions are played. Values in range 0.0 to 1.0 are slower, and values greater than 1.0 are faster."

Reply:
Actually, I see this is the other way around- higher values are slower. It might make more sense to users to invert the scale? Well, we should at least document which way is which.

Reply:
I think we could map it and make it an enum: "very-slow" "slow" "normal" "fast" "very-fast"
this would reach from 0.5 to 1.5
higher or lower values make no sense... because it is almost incomprehensible